presto-main/src/main/java/io/prestosql/operator/aggregation/builder/SpillableHashAggregationBuilder.java

```diff
@@ -155,8 +155,7 @@ private boolean hasPreviousSpillCompletedSuccessfully()
     public ListenableFuture<?> startMemoryRevoke()
     {
         checkState(spillInProgress.isDone());
-        spillToDisk();
-        return spillInProgress;
+        return spillToDisk();
```

Comment:
also remove `checkState(spillInProgress.isDone());` above (this should be, and it is, checked in `spillToDisk` method)